src/css/Preview.css

```diff
@@ -0,0 +1,41 @@
+#preview {
+  height: calc(100vh - 180px);
+  overflow-x: hidden;
+  overflow-y: auto;
+}
+
+#image {
+  margin: 0;
+  padding: 0;
+  max-width: unset;
+  max-height: unset;
```

Comment:
what does this do?

Reply:
it resets any value max-width has if it is defined somewhere else

Reply:
The grid component was acting up a lot and it made it very narrow without this, as it for some reason set a max-width of like 400px, when i needed it to be fully responsive.